### ISSUE TITLE
draw_tree error, adata.uns['schist']['state'] is not assigned

### ISSUE THREAD

**Issue by neurOnur** [dawe/schist#55]
schist runs successfully, including the `scs.inference.nested_model(adata)` function. However, as a result `adata.uns['schist']['state']` is not assigned, and `scs.plotting.draw_tree(adata)` give the error `KeyError: 'state'`.
Going back to the code, I can see that [nested_model](https://github.com/dawe/schist/blob/master/schist/inference/_nested_model.py) lists `adata.uns['schist']['state']` as an output, however, this is never assigned later in the file.

I also saw a previous issue "Remove state from unstructured data and IO functions[ #12](https://github.com/dawe/schist/issues/12)" where 'state' is removed from adata.uns['schist']. A relevant bug is fixed in version 0.7.11 that I am using right now. The problem is that [pl.draw_tree()](https://github.com/dawe/schist/blob/master/schist/plotting/_draw_tree.py) still requires the 'state' (pls see in line 83). 

I am not sure how to proceed or if I am totally missing the point. Could the producers confirm that the package is running well for them?

**Comment by dawe** [comment]
Hi, while waiting for fix this, you may try a workaround that requires generating a block_state and putting it into your dictionary. Something like

```
adata.uns['schist']['state'] = schist.tools._tl_utils.state_from_blocks(adata)

```

should work

**Comment by neurOnur** [comment]
It indeed does, thanks a lot! The visualisation is stunning

**Comment by dawe** [comment]
Should be fixed in 0.7.12
